### PR TITLE
DBZ-2660 Add eventhubs to server distro

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-2660

### CODE REVIEW

File: debezium-server/debezium-server-dist/pom.xml [hunk 1]
```diff
@@ -79,6 +79,10 @@
                     <groupId>io.debezium</groupId>
                     <artifactId>debezium-server-pulsar</artifactId>
                 </dependency>
+                <dependency>
```
Comment (gunnarmorling) [inline debezium-server/debezium-server-dist/pom.xml:82]: License-wise it's so we can distribute all the deps?

**Review by gunnarmorling — COMMENTED** [review]
@jpechane, one question inline.

File: debezium-server/debezium-server-dist/pom.xml (hunk 1 above)
Comment (jpechane) [reply debezium-server/debezium-server-dist/pom.xml:82]: Yes, as I looked it seems to be MIT. IIRC I've already went through it, just this step was forgotten in the original PR.

**Comment by gunnarmorling** [comment]
Ok, thx! // CC @abhirockzz, this will be part of the upcoming Debezium 1.4.0.Alpha1 release.

**Pull request merged**